curator-framework/src/main/java/org/apache/curator/framework/imps/GetChildrenBuilderImpl.java

```diff
@@ -165,14 +165,15 @@ public void performBackgroundOperation(final OperationAndData<String> operationA
                 @Override
                 public void processResult(int rc, String path, Object o, List<String> strings, Stat stat) {
                     watching.commitWatcher(rc, false);
+                    if (strings == null) {
+                        strings = Lists.newArrayList();
+                    }
```

Comment:
```suggestion
```

Reply:
I'm actually reverting this change, since I was just moving the call up a few lines. When creating the `CuratorEventImpl`, `strings` was already made to be non-null, so I didn't want to undo that.

Reply:
I saw that. Thank you!